CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
+## 0.2.3
 * Element Type nodes - introduces an ElementType category to the Dynamo library
+* Added a large number of new Nodes - Group nodes, Space nodes, Elevation marker nodes, Dimension nodes, Area nodes, View port nodes, Family document nodes, Family instance nodes, Warning nodes.
```

Comment:
Do not use "a large number" here, could you give more accurate number of nodes?

Reply:
OK, I will update it.